src/EditorFeatures/Core/Implementation/Classification/SyntacticClassificationTaggerProvider.TagComputer.cs

```diff
@@ -216,7 +200,8 @@ private async Task EnqueueParseSnapshotWorkerAsync(Document document, Cancellati
                 }
 
                 // preemptively parse file in background so that when we are called from tagger from UI thread, we have tree ready.
```

Comment:
A comment here that in the F#/TypeScript case we're are expecting this to be null is appropriate. I appreciate your use of discard here so it's clear we don't want that.

(I also admit I wonder if this is useful -- the background parser would already be parsing this too...)

Reply:
I also wonder if it's useful.  Isn't the SyntaxTree held by a `WeakReference`, so there is a decent chance that it will just be collected anyway?